Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,44 @@
+public class View: Viewable {
+	var ui: Receivable 
+	var guess: String!
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
+		guess = ui.getUserInput()
+		if (containsOnlyLetters(guess)) {
+			return guess
+		} else {
+			print("Invalid guess. Please enter a lowercase letter or word.")
+			receiveGuess()
+		}
+		return guess
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) -> String {
+		var blanks: [String] = []
+		let letters = gameWord.characters.map { String($0) }
+		
+		for letter in letters { 
+			if (correctGuesses.contains(letter)) { 
+           		blanks.append(letter)  
+       		} else { 
+           		blanks.append("__ ") 
+       		} 
+   		}
+   		let assignedBlanks = blanks.joinWithSeparator(" ")
+   		ui.display(assignedBlanks)
+   		return assignedBlanks
```

Comment:
I am going to say you think no. I, of course, agree with you because you know more than me, however I am not sure how to test this method if it is printing and not returning something... I will try to figure something out but for now, it is a working, tested method.